configurations/development/docker-compose.yml

```diff
@@ -16,16 +16,106 @@
 version: '2'
 
 services:
-  carma:
-    image: usdot-carma/carma
+  roscore:
+    image: usdotfhwastol/carma-base:2.8.3
```

Comment:
Similarly, can we use "usdotfhwastol/carma:latest" here?

Reply:
Yes, and the same below.